applications/GeoMechanicsApplication/tests/cpp_tests/test_apply_constant_phreatic_multi_line_pressure_process.cpp

```diff
@@ -129,14 +129,15 @@ KRATOS_TEST_CASE_IN_SUITE(ApplyConstantPhreaticMultiLinePressureProcessDoesNotTh
                 "y_coordinates": [0.0, 1.0],
                 "z_coordinates": [0.0, 0.0],
                 "gravity_direction": 1,
-                "out_of_plane_direction": 2
+                "out_of_plane_direction": 2,
+                "table": [0, 0]
```

Comment:
I would expect the 'constant' process doesn't need a table here

Reply:
Indeed, reverted the wrong addition.